.devcontainer/devcontainer.json

```
@@ -13,13 +13,11 @@
     "ghcr.io/devcontainers/features/node:1": {
       "version": "lts"
     },
+    "ghcr.io/devcontainers-extra/features/nx-npm:1": {},
```

Would be good to pin this global installation to the workspace version:

```suggestion
    "ghcr.io/devcontainers-extra/features/nx-npm:1": {
         "version": "19.0.4"
    },
```